src/interfaces/WF/WFIPAddressAddress.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @typedef {('External'|'Local')} WFIPAddressSourceOption
```

Comment:
Could you remove this `typedef` please? See my other comment for a bit more info 👍